docker-compose.yml

```diff
@@ -39,8 +39,10 @@ services:
       PS_DEV_MODE: ${PS_DEV_MODE:-0}
       ADMIN_MAIL: ${ADMIN_MAIL:-demo@prestashop.com}
       ADMIN_PASSWD: ${ADMIN_PASSWD:-Correct Horse Battery Staple}
+      XDEBUG_MODE: ${PS_XDEBUG_MODE:-off}
```

Comment:
Don't we need our images to contain xdebug beforehand?

Reply:
Yes, we would have to expand the images to install xdebug